src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -97,51 +101,114 @@ class AmendmentTableImpl : public AmendmentTable
     amendmentList_t getDesired();    // amendments we support, do not veto, are not enabled
 };
 
-void
-AmendmentTableImpl::addInitial ()
+namespace detail
```

Comment:
detail namespaces are for header files not for .cpp files. The contents of .cpp files are already implicitly private, identifiers do not need additional markings. A detail namespace is used when it is necessary for public interface declarations in headers to have visibility on private identifiers. This said, there's no harm using a detail namespace in a .cpp file but it adds no value.